src/ast/nodes/FunctionExpression.ts

```diff
@@ -25,6 +26,14 @@ export default class FunctionExpression extends FunctionNode {
 		return super.parseNode(esTreeNode);
 	}
 
+	protected onlyFunctionCallUsed(): boolean {
+		const isIIFE =
+			this.parent.type === NodeType.CallExpression &&
+			(this.parent as CallExpression).callee === this &&
+			!this.id;
+		return isIIFE || super.onlyFunctionCallUsed();
+	}
```

Comment:
so it looks like the functional change here is making `isIIFE` compute to false for named functions?

(slightly better would be to check that the function value is referenced anywhere, I don't know how easy/quick that is, right?)

Reply:
Yeah we can check it carefully to support to  no reference or recursion.